app/grandchallenge/challenges/models.py

```diff
@@ -411,13 +411,11 @@ class Challenge(ChallengeBase, FieldChangeMixin):
         on_delete=models.PROTECT,
         related_name="external_evaluators_of_challenge",
     )
-    forum = deprecate_field(
-        models.OneToOneField(
-            machina_forum_models.Forum,
-            null=True,
-            editable=False,
-            on_delete=models.PROTECT,
-        )
+    forum = models.OneToOneField(
```

Comment:
Can't deprecate the field just yet since the migration and management command reference the forum field.